ModeloPersona.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjerciciooUno
+{
+    internal class ModeloPersona
+    {
+
+        // se crean las variables privadas para luego agregar Set y get
+
+        private string colorPelo;
+        private int edad;
+        private string nombre;
+
+
+        // se crean Get y Set
+
+        public string ColorPelo { get => colorPelo; set => colorPelo = value; }
```

Comment:
This is not correct:

1. A variable can not be name with upper case at the beggining
2. You are creating two variable colorPelo and ColorPelo and that is not correct you have to create just one
3. To do the encapsulation on this you have to use the private variable for example 
For the variable colorPersona which is private you can create two method one to set the informatino and get the get the information from that variable, example:
```

        private string colorPelo;


        public String getColorPelo(){
            return colorPelo;
        }

        public setColorPelo(String colorPelo) {
             this.colorPelo = colorPelo;
        }
```

Reply:
So delete those variables from line 21 to 23 and use just the attributes created at the beginning

Reply:
has been corrected